tools/code_format/check_format.py

```diff
@@ -895,6 +895,15 @@ def clangFormat(file_path):
     return ["clang-format rewrite error: %s" % (file_path)]
   return []
 
+# Fix for https://github.com/envoyproxy/envoy/issues/10535
+def checkProtoiValidation(file_path):
```

Comment:
Can you add a test in https://github.com/envoyproxy/envoy/blob/master/tools/code_format/check_format_test_helper.py for this behavior?

Reply:
@htuch Hi, i have one doubt here. i found that test file path ("testdata/check_format/") in script "check_format_test_helper.py"   are defined as excluded path in script file "tools/code_format/check_format.py" , which causing test to deviate from actual behavior as imposed in script file. 
please suggest way-forward.

Reply:
@htuch sorry, it was the issue with my local env. now it is fixed.